tuxemon/core/components/event/actions/combat.py

```diff
@@ -213,6 +213,9 @@ def random_encounter(self, game, action):
         if not self.check_battle_legal(player1):
             return False
 
+        #stop player motion
+        player1.moving = False
```

Comment:
The problem with simply setting the player moving to `False` is that it will stop the player mid-step between tiles. After a few encounters the player will start to become offset from the tiles like this:

![](http://i.imgur.com/c58GqEO.png)